docs/source/examples/multipleROIs.py

```diff
@@ -88,30 +90,33 @@ def plot_roi_data(tag):
     width_dim.label = "width"
     color_dim = data.append_set_dimension()
     color_dim.labels = channels
-    
+
     # some space for three regions-of-interest
-    roi_starts = np.zeros((3,3))
-    roi_starts[0, :] = [250, 245, 0] 
-    roi_starts[1, :] = [250, 315, 0] 
-    roi_starts[2, :] = [340, 260, 0] 
-
-    roi_extents = np.zeros((3,3))    
-    roi_extents[0, :] = [30, 45, 3] 
-    roi_extents[1, :] = [30, 40, 3] 
-    roi_extents[2, :] = [25, 65, 3] 
-        
+    roi_starts = np.zeros((3, 3))
+    roi_starts[0, :] = [250, 245, 0]
+    roi_starts[1, :] = [250, 315, 0]
+    roi_starts[2, :] = [340, 260, 0]
+
+    roi_extents = np.zeros((3, 3))
+    roi_extents[0, :] = [30, 45, 3]
+    roi_extents[1, :] = [30, 40, 3]
+    roi_extents[2, :] = [25, 65, 3]
+
     # create the positions DataArray
-    positions = block.create_data_array("ROI positions", "nix.positions", data=roi_starts)
-    positions.append_set_dimension() # these can be empty
+    positions = block.create_data_array(
+        "ROI positions", "nix.positions", data=roi_starts)
+    positions.append_set_dimension()  # these can be empty
     positions.append_set_dimension()
-    
+
     # create the extents DataArray
-    extents = block.create_data_array("ROI extents", "nix.extents", data=roi_extents)
+    extents = block.create_data_array(
+        "ROI extents", "nix.extents", data=roi_extents)
```

Comment:
As I see it PEP is a suggestion and this line (and 106, 107) are examples were to bend the rules :) this is quite ugly...

If you need to wrap the line, do it before the last argument respectively the last arguments that are beyond the 80 character threshold.